docs/api/rest_api/rest_api_reference/input/ez-cart.raml

```diff
@@ -6,8 +6,8 @@ post:
         Content-Type:
             description: The CartCreateStruct schema encoded in XML or JSON format.
```

Comment:
```suggestion
            description: The CartCreate schema encoded in XML or JSON format.
```

Reply:
Not sure about that, as in fact `CartCreateStruct` object is passed to the API.

Reply:
Changed to avoid confusion given the example from https://github.com/ibexa/documentation-developer/pull/2172/commits/4c2a6188f7efee8eb7f7576019ce43e2081d8755.

Reply:
@konradoboza  I guess it is supposed to match the payload schema, not what the parser will transform the payload into.

Reply:
See https://github.com/ibexa/documentation-developer/pull/2172#discussion_r1356775261.